api/tools/src/main/java/org/pmiops/workbench/tools/CommandLineToolConfig.java

```diff
@@ -34,6 +34,10 @@
  *
  * <p>The main difference is that request-scoped dependencies cannot be used from a command-line
  * context.
+ *
+ * <p>IMPORTANT: This config should only be used in a database context. In order to implement a tool
```

Comment:
Can we set it up so it fails to inject when that condition isn't met?

Reply:
@jaycarlton I'm not aware of any addons that do this

Reply:
OK. I know emacs can do it. I'll find one.

Even better if we could tweak spotless to do it for us.